modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/CommentResourceImpl.java

```diff
@@ -728,13 +737,15 @@ private Comment _postParentCommentComment(
 			String className, long classPK, String text)
 		throws Exception {
 
-		return _postComment(
-			() -> _commentManager.addComment(
-				externalReferenceCode, _getUserId(), className, classPK,
-				StringPool.BLANK, parentCommentId, StringPool.BLANK,
-				StringBundler.concat("<p>", text, "</p>"),
-				_createServiceContextFunction()),
-			className, classPK, groupId);
+		return CommentUtil.toComment(
+			_commentManager.addComment(
+				() -> _commentManager.addComment(
```

Comment:
Same thing here, this is a little strange, an addComment method calling another addComment method, let's see if we can improve this in some way.